### ISSUE TITLE
jvm arguments

### ISSUE THREAD

**Issue by jakob-grabner** [mannodermaus/android-junit5#32]
I try to run robolectric test with junit5 and run into an issue. Because of some library I use I need to a `-noverify` to the jvm arguments. Previously (with junit4) this worked like this:
```
testOptions {
        unitTests {
            all {
                // configure the test JVM arguments
                jvmArgs '-noverify'
            }
        }
    }
```
But unfortunately this does not work with junit5 and this plugin. I'm not sure, but it seems the plugin does not use the provided jvm arguments.

**Comment by mannodermaus** [comment]
Thanks for reporting! The Android Gradle Plugin's `testOptions.unitTests` clause applies to all Gradle tasks of type `Test`, which are confined to running JUnit 3.8 & 4 as well as TestNG tests. Therefore, the default JUnit 5 plugin (and by extension, this one) utilize a `JavaExec` task which invokes the `ConsoleLauncher`. Unfortunately, this means that params like `jvmArgs` aren't passed through to these tasks until Gradle provides a proper common interface for JUnit 5 tasks, and the Android Gradle Plugin honors this interface.

However, since we're already using a lot of work from the default tasks (i.e. classpath collection), I don't see why it wouldn't be possible to add more configuration to that, though. I've created some failing test cases to develop against, so it will become possible to use `jvmArgs` and `systemProperties` from inside `testOptions.unitTests.all` and have the respective values used in JUnit 5 tasks as well. I'll update once a SNAPSHOT release is available for this.

**Comment by mannodermaus** [comment]
This will be included in `1.0.21` and the upcoming first version for JUnit 5.1.0-M1. You can try it using the `1.0.21-SNAPSHOT`!

**Comment by mannodermaus** [comment]
Released in 1.0.21

**Comment by jakob-grabner** [comment]
Awesome thx!
